test/uk/gov/hmrc/tai/service/IncomeServiceSpec.scala

```diff
@@ -141,7 +141,108 @@ class IncomeServiceSpec extends PlaySpec with MockitoSugar {
           val result = Await.result(SUT.updateTaxCodeIncome(nino, taxYear,1,1234)(HeaderCarrier()), 5 seconds)
 
           result mustBe IncomeUpdateSuccess
-          verify(mockAuditor).sendDataEvent(Matchers.eq("Update Multiple Employments Data"), any())(any())
+
+          val auditMap = Map("nino" -> nino.value,
+            "year" -> taxYear.toString,
+            "employmentId" -> "1",
+            "newAmount" -> "1234",
+            "currentAmount" -> "123.45")
+          verify(mockAuditor).sendDataEvent(Matchers.eq("Update Multiple Employments Data"), Matchers.eq(auditMap))(any())
+        }
+
+        "the current amount is not provided due to no incomes returned" in {
+
+          val taxYear = TaxYear()
+
+          val mockEmploymentSvc = mock[EmploymentService]
+          when(mockEmploymentSvc.employment(any(), any())(any()))
```

Comment:
I think we should be moving towards using less `any()` and more specific params that we expect. 

Ive been hit before where I was passing a incorrect taxYear and the specs wouldn't fail because the mock was looking for an any()